donate/settings/base.py

```diff
@@ -37,6 +37,10 @@ class Base(object):
     # Basket Configuration
     BASKET_API_ROOT_URL = env('BASKET_API_ROOT_URL')
     BASKET_SQS_QUEUE_URL = env('BASKET_SQS_QUEUE_URL')
+    # Basket client configuration
+    BASKET_URL = env('BASKET_URL')
+    BASKET_API_KEY = env('BASKET_API_KEY')
```

Comment:
Now that we know more about the Basket client library, and that it doesn't require authentication, do you think we'll ever need to "use the lookup_user method with an email address"? (taken from basket client docs)

 I can't see any reason we'd need to do that in the near future, so I think it's safe to remove the BASKET_API_KEY.
```suggestion
```